packages/ruby/lib/readme/metrics.rb

```diff
@@ -1,8 +1,17 @@
 require "readme/metrics/version"
+require "httparty"
 
 module Readme
-  module Metrics
-    class Error < StandardError; end
-    # Your code goes here...
+  class Metrics
+    def initialize(app, endpoint)
+      @app = app
+      @endpoint = endpoint
+    end
+
+    def call(env)
+      path = "#{env["REQUEST_METHOD"]} #{env["PATH_INFO"]}"
```

Comment:
Might not be worth doing right now, but I could imagine some sort of builder or decorator for `env` that rolls this into an object that's easier to test

Reply:
Yes, there will definitely be an object extracted here since the real logic is somewhat complex.